js/script.js

```diff
@@ -103,31 +112,36 @@ function scaleToFit(img) {
     var y = (canvas.height / 2) - (img.height / 2) * scale;
     ctx.drawImage(img, 0, 0, img.width * scale, img.height * scale);
 }
-
+/**
+ * Returns None
+ * Draws and animates the scrolling grass platform
+ */
 function drawPlatform() {
-    // reset, start from beginning
-    if (pfX < -pf.width) {
+    // If platform x-coordinate is <= width of platform image width then reset x-coordinate
+    if (pfX <= -pf.width) {
         pfX = 0;
     }
-    // draw additional image
+    // Platform image width is smaller than canvas width so we must draw a second platform
     if (pfX < canvas.width - pf.width) {
         ctx.drawImage(pf, pfX + pf.width, pfY, pf.width, pf.height);
     }
-    // draw image
+    // Draw initial platform
     ctx.drawImage(pf, pfX, pfY, pf.width, pf.height);
-    // amount to move
+    // Distance x-coordinate of platform is moved
     pfX += dx;
 }
-
+/**
+ * Returns None
+ * Draws chef Kirby, walking Kirby
```

Comment:
Doesn't this draw the mirror with the currentmeme?

Reply:
Oops I will edit this comment.